src/react-components/src/__tests__/components/FormDropdown-test.tsx

```diff
@@ -16,20 +16,36 @@ describe("FormDropdown", () => {
     changeCallback = stub();
     blurCallback = stub();
   });
+  it("Form Dropdown should render a FormDropdown", () => {
+    wrapper = Enzyme.shallow(<FormDropdown labelId="label" isRequired={false} dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
+    expect(wrapper.find("select")).to.have.lengthOf(1);
+  });
 
   it("requires either label or aria-label", () => {
-    expect(() => Enzyme.mount(<FormDropdown dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />))
-    .to.throw("Either labelText or ariaLabel must be defined");
+    expect(() => Enzyme.mount(<FormDropdown isRequired={false} dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />))
+      .to.throw("Must either have labelId or aria-label");
   });
 
-  it("Form Dropdown should render a FormDropdown", () => {
-    wrapper = Enzyme.shallow(<FormDropdown dropdownId="hi" labelText="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
-    expect(wrapper.find("select")).to.have.lengthOf(1);
+  it("if both labelId and ariaLabel are defined, select has aria-labeledBy", () => {
+    wrapper = Enzyme.shallow(<FormDropdown labelId="label" ariaLabel="aria-label" isRequired={false} dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
+    expect(wrapper.find("select").props()["aria-labelledby"]).to.equal("label");
+  });
+
+  it("if only aria-label is defined, select has aria-label", () => {
+    wrapper = Enzyme.shallow(<FormDropdown ariaLabel="aria-label" isRequired={false} dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
+    expect(wrapper.find("select").props()["aria-label"]).to.equal("aria-label");
+  });
+
+  it("Form Dropdown should render an icon", () => {
+    wrapper = Enzyme.shallow(<FormDropdown labelId="label" isRequired={false} dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
+    expect(wrapper.find("Icon")).to.have.lengthOf(1);
   });
 
   it("sends callback with value on change", () => {
     wrapper = Enzyme.shallow(<FormDropdown
-      dropdownId="hi" labelText="hi"
+      labelId="label"
+      isRequired={false}
+      dropdownId="hi"
```

Comment:
Purely stylistic: some of these calls are either on one long line, missing some indentation, or in this "one line per prop". I prefer the latter and would stick with one just for consistency, but again, this is not detrimental to the PR and can be done post-MVP. Maybe introducing [prettier](https://prettier.io/) might be helpful later.

Reply:
Seconding prettier. [Added a card](https://github.com/NYPL/nypl-design-system/projects/1#card-31366341) to our backlog.